plugin/src/types.ts

```diff
@@ -13,6 +13,7 @@ export interface ReanimatedPluginOptions {
   omitNativeOnlyData?: boolean;
   globals?: string[];
   substituteWebPlatformChecks?: boolean;
+  autoworkletizationRequests?: Record<string, string[]>;
```

Comment:
Let's rename it with a verb to match other props, e.g. `omitNativeOnlyData`, `substituteWebPlatformChecks`
```suggestion
  workletizeFiles?: Record<string, string[]>;
```
Also, if the files need to be listed explicitely, it's no longer automatic